traffic_ops/testing/api/v4/regions_test.go

```diff
@@ -210,8 +210,12 @@ func SortTestRegionsDesc(t *testing.T) {
 	for start, end := 0, len(respDesc)-1; start < end; start, end = start+1, end-1 {
 		respDesc[start], respDesc[end] = respDesc[end], respDesc[start]
 	}
-	if respDesc[0].Name != respAsc[0].Name {
-		t.Errorf("Regions responses are not equal after reversal: Asc: %s - Desc: %s", respDesc[0].Name, respAsc[0].Name)
+	if len(respDesc[0].Name) > 0 && len(respAsc[0].Name) > 0 {
+		if respDesc[0].Name != respAsc[0].Name {
+			t.Errorf("Regions responses are not equal after reversal: Asc: %s - Desc: %s", respDesc[0].Name, respAsc[0].Name)
+		}
+	} else {
+		t.Errorf("Region name shouldn't be empty while sorting the response")
```

Comment:
This change seems unrelated to this PR

Reply:
Just added null checks, and PR related to theis functionality has closed. Do you want me to include these changes in the respective PR?